iconrpcserver/utils/message_queue/channel_inner_stub.py

```diff
@@ -34,31 +34,37 @@ class ChannelInnerTask:
         pass
 
     @message_queue_task
-    async def get_block_v2(self, block_height, block_hash, block_data_filter, tx_data_filter) -> Tuple[int, str, str, list]:
+    async def get_block_v2(self, block_height, block_hash, block_data_filter, tx_data_filter) \
```

Comment:
```suggestion
    async def get_block_v2(self,
        block_height, block_hash,
        block_data_filter,
        tx_data_filter) -> Tuple[int, ...]:
```
I suggest to use break parameter in multiline.
Please reference this [mypy](https://github.com/python/mypy/blob/master/mypy/binder.py#L343) link.

Reply:
That would be more organized. Thanks for suggestion!